test/css-local-import/main.js

```diff
@@ -0,0 +1,12 @@
+require("./main.css");
+
+var p = document.getElementsByTagName("p").item(0);
+var s = getComputedStyle(p);
+
+if (window.assert) {
+	assert.equal(s.fontSize, "30px", "should inline the @import");
```

Comment:
need to update this text, we are no longer inlining the import, but since clean-css does no longer removes them. It should still work.